ui/lib/core/addon/components/selectable-card.hbs

```diff
@@ -0,0 +1,52 @@
+{{#if @hasSearchInput}}
+  <div class="selectable-card is-rounded no-flex" data-test-selectable-card-form={{@cardTitle}}>
+    <div class="is-flex-between is-fullwidth card-details">
+      <h3 class="title is-5" data-test-selectable-card>{{@cardTitle}}</h3>
+    </div>
+    <div class="has-top-bottom-margin">
+      <p class="sub-text">{{@subText}}</p>
+    </div>
+    <form class="is-flex" data-test-selectable-card={{@cardTitle}}>
+      <SearchSelect
```

Comment:
I decided not to `{{yield}}` here and instead put the form inputs in the child component since this `SelectableCard` will only have 1 input and button. It also causes a lot of repetition in the parent component since the parent will have to have a `handleInput` and `tracked` property for each input field.

Reply:
I'm not quite following the logic to hard-code the `SearchSelect` into this component. To me, especially since this will live in our `core` addon library, it should be as generic as possible and this one particularly feels like it should only have layout concerns. 
If we do want to keep SearchSelect coded into this component for some reason, I would suggest we move the `{{#if}}` block within the layout divs so that we don't have to duplicate all the code, and definitely write a test handling the new use case

Reply:
I might have added confusion yesterday when Kianna and I synced. I thought we had landed on `{{yield}}`-ing any inputs. I'm sorry if I was unclear!

Now that this component has been moved to the `core` addon, I agree it makes sense to think of the SelectableCard as the style template that can yield any input (For example, a future use case may include both a top right link and input).

Reply:
No problem! Learned a lot!! Ty!!